pkgs/tools/archivers/peazip/default.nix

```diff
@@ -0,0 +1,93 @@
+{ mkDerivation
+, lib
+, fetchFromGitHub
+, wrapQtAppsHook
+, fpc
+, lazarus
+, xorg
+, libqt5pas
+, _7zz
+, archiver
+, brotli
+, upx
+, zpaq
+, zstd
+}:
+
+mkDerivation rec {
+  pname = "peazip";
+  version = "8.6.0";
+
+  src = fetchFromGitHub {
+    owner = "peazip";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-MF0MwIKV/25lm+IKRdFXSMiXSQxTKksGBj/r0td+v/o=";
+  };
+  sourceRoot = "source/peazip-sources";
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+    lazarus
+    fpc
+  ];
+
+  buildInputs = [
+    xorg.libX11
+    libqt5pas
+  ];
+
+  NIX_LDFLAGS = "--as-needed -rpath ${lib.makeLibraryPath buildInputs}";
+
+  buildPhase = ''
+    HOME=$(mktemp -d) lazbuild --lazarusdir=${lazarus}/share/lazarus --widgetset=qt5 --build-all project_pea.lpi && [ -f pea ]
+    HOME=$(mktemp -d) lazbuild --lazarusdir=${lazarus}/share/lazarus --widgetset=qt5 --build-all project_peach.lpi && [ -f peazip ]
```

Comment:
Is there any reason why not single `export HOME=...`?

Reply:
Maybe post processes should have a seperate `HOME`? I don't know if they should…